scipy/stats/tests/test_stats.py

```diff
@@ -5037,6 +5039,31 @@ def test_location_sign(self, ksfunc, alternative,
         assert res.statistic_location == ref_location
         assert res.statistic_sign == ref_sign
 
+@make_xp_test_case(stats.ttest_rel)
+def test_ttest_rel_xp(xp):
+    # stats.ttest_rel had no tests using the xp fixture, which causes a failure
+    # in tools/check_xp_untested.py. A portion of the below test has been
+    # converted as a temporarily measure. TODO: Convert all of test_ttest_rel
+    # and refactor it into separate tests.
```

Comment:
That's because the entire implementation of `ttest_rel` is:
```python3
    return ttest_1samp(a - b, popmean=0., axis=axis, alternative=alternative,
                       _no_deco=True)
```
It's basically just an alias now.
So if `ttest_1samp` is adequately tested, so is `ttest_rel`. 
I think it's worth skimming the `ttest_rel` tests for unique ideas and adding any to the `ttest_1samp` tests, but I think we can let go of the rest.

Then, if needed, I think we could run all the `ttest_1samp` on a wrapper function that calls `ttest_rel` because - numerical precision details aside - tests of these three things are equivalent: 
```python3
ttest_1samp(x, popmean)
ttest_1samp(x - popmean, 0)
ttest_rel(x - popmean + y, y)  # y is random of the same shape of x
```